samples/light_control/src/core.c

```diff
@@ -9,7 +9,7 @@
 
 #include <zephyr.h>
 #include "zeta.h"
-#include "zeta_callbacks.h"
+#include "zeta_services.h"
```

Comment:
I was wondering the possibility of including zeta_services.h into zeta.h. What do you think? Doing this, the user only needs to include zeta.h

Reply:
I think that it is a good idea. This could be a new issue, where in addition we need to change zeta_custom_functions.h name to something related to channels. What do you think?